test/unit/uk/gov/hmrc/tariffclassificationfrontend/utils/DatesSpec.scala

```diff
@@ -14,12 +14,11 @@
  * limitations under the License.
  */
 
-package unit.uk.gov.hmrc.tariffclassificationfrontend.utils
+package uk.gov.hmrc.tariffclassificationfrontend.utils
```

Comment:
revert

Reply:
Seems a bit weird though, the test should be in the same package as the thing it's testing. We have the ability to create different test source sets though

Reply:
It is a personal preference.
But I like the idea of having all tests inside a single folder (`test`)

Reply:
![image](https://user-images.githubusercontent.com/4864094/47155065-e10a0080-d2db-11e8-85d1-14796d878fc3.png)
This is how it looks now